boards/arm/ebyte_e73_tbb_nrf52832/ebyte_e73_tbb_nrf52832.dts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2017 Shawn Nock <shawn@monadnock.ca>
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2021 Michal Morsisko
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52832_qfaa.dtsi>
+#include "ebyte_e73_tbb_nrf52832-pinctrl.dtsi"
+
+/ {
+	model = "EBYTE E73-TBB NRF52832";
+	compatible = "ebyte,e73-tbb-nrf52832";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 17 GPIO_ACTIVE_LOW>;
+			label = "Red LED1";
+		};
+		led1: led_1 {
+			gpios = <&gpio0 18 GPIO_ACTIVE_LOW>;
+			label = "Red LED2";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_led0: pwm_led_0 {
+			pwms = <&pwm0 17>;
```

Comment:
this needs to be fixed: pwm now uses channel not pin, and it also requires period + flags, see other nRF boards.

Reply:
Done